index.vwf.config.yaml

```diff
@@ -23,6 +23,6 @@ view:
   vwf/view/threejs: "#vwf-root"
   vwf/view/blockly: { "blocklyPath": './source/blockly/', "toolbox": 'toolbox' }
   vwf/view/sound:
-  # vwf/view/editor:
+  vwf/view/editor:
```

Comment:
Re-disable this. We never want the editor enabled on a public version of the game.